src/apps/frontend/utils/use-timer.hook.ts

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+
+import { OTP_RESEND_DELAY_IN_MILLISECONDS, MILLISECONDS_IN_A_SECOND } from '../constants/timer';
+
+type UseTimerType = {
+  isResendEnabled: boolean;
+  remaininingSecondsStr: string;
+  startTimer: () => void;
+  stopTimer: () => void;
+};
+
+const useTimer = (): UseTimerType => {
```

Comment:
Why does user time hook knows anything about OTP? For example, I should be able to use this hook at other places beyond otp component